src/scenes/home/contact/contact.js

```diff
@@ -8,16 +8,13 @@ const Contact = () => (
   <div>
     <Section title="Contact Us">
       <p className={styles.centerText}>
-        We are a decentralized community of hard-working volunteers, and we love hearing feedback,
-        comments, and suggestions!
+        We are a decentralized community of hard-working volunteers,
+        <n /> and we love hearing feedback, comments, and suggestions!
```

Comment:
@christiandavidturner What's the purpose of the `<n/ >` tag? Was it to get around the ES-Lint `max-len` rule limiting lines to 100 characters?

 `<n/ >` is not an HTML element, nor a custom React component, so I'd prefer it not be in the codebase - this could be confusing to others.

Can you please remove the `<n />` (and keep the same line formatting you have above)? The code will still pass linting and output the correct HTML.

Reply:
Yeah, that's exactly why. I just changed it back!